app/controllers/items_controller.py

```diff
@@ -0,0 +1,230 @@
+        api.abort(HTTPStatus.NOT_FOUND, f'Item {data["item_id"]} not found')
+
+@api.route('/me/storage/<int:storage_id>')
+@api.doc(security='access-token')
+class ItemsMeByStorageIdApi(Resource):
+    @api.doc('get_items_by_storage_me')
+    @api.response(200, 'Success', [ItemsModels.item])
+    @api.response(401, 'Unauthorized')
+    @api.response(403, 'Forbidden operation')
+    @api.response(404, 'Not found')
+    @token_required
+    def get(self, storage_id: int):
+        """Get items from own storage"""
+        user = get_user_from_request(api)
+        storage = Storages.get_storage_by_id(storage_id)
+        if storage is not None:
+            if user['user_id'] == storage['user_id']:
+                return Items.get_items_by_storage(storage['storage_id'])
+            api.abort(HTTPStatus.FORBIDDEN, f'Storage {storage_id} is not yours')
+        api.abort(HTTPStatus.NOT_FOUND, f'Storage {storage_id} not found')
+
+
+@api.route('/me/<int:item_id>')
+@api.doc(security='access-token')
+class ItemsMeByIdApi(Resource):
+    @api.doc('delete_me_item_by_id', security='access-token')
+    @api.response(204, 'Item deleted')
+    @api.response(401, 'Unauthorized')
+    @api.response(403, 'Forbidden operation')
+    @api.response(404, 'Not found')
+    @token_required
+    def delete(self, item_id: int):
+        """Delete item in own storage by id"""
+        user = get_user_from_request(api)
+        item = Items.get_item_by_id(item_id)
+        if item is not None:
+            storage = Storages.get_storage_by_id(item['storage_id'])
+            if user['user_id'] == storage['user_id']:
+                item_response = Items.delete(item_id)
+                if item_response:
+                    return '', 204
+                api.abort(HTTPStatus.FORBIDDEN, f'Item {item_id} can\'t be deleted')
+            api.abort(HTTPStatus.FORBIDDEN, f'Item {item_id} is not yours')
+        api.abort(HTTPStatus.NOT_FOUND, f'Item {item_id} not found')
+
+
+@api.route('/storage/<int:storage_id>')
+@api.doc(security='access-token')
+class ItemsMeByStorageIdApi(Resource):
+    @api.doc('get_items_by_storage')
+    @api.response(200, 'Success', [ItemsModels.item])
+    @api.response(401, 'Unauthorized')
+    @api.response(403, 'Forbidden operation')
+    @api.response(404, 'Not found')
+    @token_required
+    def get(self, storage_id: int):
+        """Get items by storage"""
+        requester = get_user_from_request(api)
+        check_admin(api, requester)
+        storage = Storages.get_storage_by_id(storage_id)
+        if storage is not None:
+            return Items.get_items_by_storage(storage['storage_id'])
+        api.abort(HTTPStatus.NOT_FOUND, f'Storage {storage_id} not found')
+
+
+@api.route('/search')
+class ItemsSearch(Resource):
+    @api.doc('search')
+    @api.expect(ItemsModels.search, validate=True)
+    @api.response(200, 'Search completed', [ItemsModels.item_public])
+    def get(self):
```

Comment:
Please, add a response for the `400` code. It is called when validation of arguments is failed.